recipes/lora_finetune_distributed.py

```diff
@@ -270,6 +276,114 @@ def setup(self, cfg: DictConfig) -> None:
             last_epoch=self.global_step - 1,
         )
 
+        # Set up profiler, returns FakeProfiler (nullcontext object with no-op `step` method)
+        # if cfg is missing profiler key or if `cfg.profiler.enabled = False`
+        self._profiler = self._setup_profiler(cfg.get(PROFILER_KEY, None), log_cfg=True)
+
+    def _setup_profiler(
+        self, cfg: DictConfig, log_cfg: bool = False
```

Comment:
nit: call this cfg_profiler, similar to the other setup methods, then it's very clear that it's the profiler field and not the full config

Reply:
Fixed